src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -78,7 +80,7 @@ def get_expected_value_range(self, timeout_ops, resent_ops):
                 ),
                 "engine_pool_xferred_fetch": (
                     ops_number * self.ior_cmd.transfer_size.value,
-                    (ops_number + timeout_ops + 1) * self.ior_cmd.transfer_size.value
+                    (ops_number + timeout_ops + 2) * self.ior_cmd.transfer_size.value
```

Comment:
Is it true that each `timeout_ops` could have been retried multiple times? If we find `timeout_ops + 2` is not sufficient, maybe this would be more flexible?
```suggestion
                    (ops_number + timeout_ops * 2) * self.ior_cmd.transfer_size.value
```

Reply:
From my observations, such value was always enough with my experiments.
I occasionally got some errors with the CI with the "leap" distribution for some obscure reasons => reasons of s/+1/+2/
At this moment, I would prefer to keep this value +2 value. I will update it according to your proposal as soon as it will fail: looking at the log in such situation should help me to better understand what is happening under the hoods.